docs/advanced/experimental/kubernetes.md

```diff
@@ -1,20 +1,68 @@
-# Jina on Kubernetes
+# Jina On Kubernetes
 
 Deploy your `Flow` on `Kubernetes`.
 ## Requirements
-Please setup a `Kubernetes` cluster and pull the credentials. e.g.:
+Please setup a `Kubernetes` cluster and pull the credentials.
```

Comment:
`pull the credentials` migth not be the correct term here? I think what we want to say is that the user has to configure k8s cluster access locally? Which most likely means setting up the k8s config so that the client can find it.

I think 'pulling the credentials' is rather GKE specific.

Reply:
Thanks, I could not find where I had this from. Changed it now.